tests/scripts/test-ref-configs.pl

```diff
@@ -30,16 +30,20 @@
 my %configs = (
     'config-ccm-psk-tls1_2.h' => {
         'compat' => '-m tls12 -f \'^TLS-PSK-WITH-AES-...-CCM-8\'',
+        'test_again_with_use_psa' => 1
     },
     'config-no-entropy.h' => {
     },
     'config-suite-b.h' => {
         'compat' => "-m tls12 -f 'ECDHE-ECDSA.*AES.*GCM' -p mbedTLS",
+        'test_again_with_use_psa' => 1,
     },
     'config-symmetric-only.h' => {
+        'test_again_with_use_psa' => 0, # Uses PSA by default, no need to test it twice
```

Comment:
This looks a bit strange. Shouldn't we have 2 flags like `use_mbedtls/use_psa`:
```
    'config-suite-b.h' => {
        'compat' => "-m tls12 -f 'ECDHE-ECDSA.*AES.*GCM' -p mbedTLS",
        'use_mbedtls' => 1,
        'use_psa' => 1,
    },
```

Reply:
I don't think we need two flags when there's only one bit of information that matters in practice: should we run this test once or twice? I think the current flags captures that accurately.